### ISSUE TITLE
Change bootstrap scripts to install Python 3.5.3

### ISSUE THREAD

**Issue by mwobensmith** [mozilla/iris2#76]
The Windows script has already been changed, but we at least need to change the Linux (and probably Mac) bootstrap to limit Python upgrade to 3.5, related to #67.

**Comment by mwobensmith** [comment]
Specifically:
* Check for Python 3.5.3
  * If not 3.5.3, install via package manager (Scoop, brew, apt-get)
* export `PATH`, `LDFLAGS`, `PKG_CONFIG_PATH`
  * Mac: Write path to `.bash_profile` if necessary
  * Win: Add path to environment variable if necessary
* `pip install --upgrade pip`
* `pip install --upgrade pipenv`

**Comment by mwobensmith** [comment]
Once this is done, we should update the Pipfile:
```
[requires]
python_version = "3.5.3"
```

**Comment by mwobensmith** [comment]
Hi @rrobotin, we have a new plan now. Sorry to go back on what we discussed earlier, but we've decided to allow Python 3.7 on Mac/Linux. We will only downgrade to 3.5.3 on WIndows 7/10. 

The bootstrap script should be adjusted for the above. Also, in the `Pipfile`, we can change the requirement for `python_version` to `>= 3.5.3`. The idea is that this will be the minimum required version of Python for all platforms, but on Windows, we will add logic to `__main__.py` to quit the run if greater than 3.5.3 (#163).
